packages/server/message-store-api/test/graph/resolver/organization.resolvers.go

```diff
@@ -26,6 +26,11 @@ func (r *mutationResolver) OrganizationDelete(ctx context.Context, id string) (*
 	panic(fmt.Errorf("not implemented: OrganizationDelete - organization_Delete"))
 }
 
+// Domains is the resolver for the domains field.
+func (r *organizationResolver) Domains(ctx context.Context, obj *model.Organization) ([]string, error) {
+	panic(fmt.Errorf("not implemented: Domains - domains"))
```

Comment:
really need to panic?

Reply:
this is auto-generated code only used to test the grapql api